org.freedesktop.LinuxAudio.Plugins.Calf.json

```
@@ -0,0 +1,61 @@
+{
+    "id": "org.freedesktop.LinuxAudio.Plugins.Calf",
+    "branch": "21.08",
+    "runtime": "org.freedesktop.LinuxAudio.BaseExtension",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk//21.08",
+    "build-extension": true,
+    "appstream-compose": false,
+    "build-options": {
+        "prefix": "/app/extensions/Plugins/Calf",
+        "prepend-ld-library-path": "/app/extensions/Plugins/Calf/lib",
+        "prepend-pkg-config-path": "/app/extensions/Plugins/Calf/lib/pkgconfig"
+    },
+    "cleanup": [
+        "/bin",
+        "/share/applications",
+        "/share/bash-completion",
+        "/share/doc",
+        "/share/icons",
+        "/share/man",
+        "/lib/lv2",
+        "/include"
+    ],
+    "modules": [
+        "shared-modules/linux-audio/fluidsynth2-static.json",
+        "shared-modules/linux-audio/lv2.json",
+        {
+            "name": "calf",
+            "config-opts": [
+                "--with-lv2",
+                "--with-lv2-dir=/app/extensions/Plugins/Calf/lv2"
+            ],
+            "build-options": {
```

```suggestion
            "build-options": {
                "ldflags": "-lpulse-simple -lpulse -lasound -ldbus-1 -lSDL2 -lreadline -Wl,--export-dynamic -lgmodule-2.0 -pthread -lglib-2.0  -lgomp",
```

This should do the trick.